metapi/metapi.js

```diff
@@ -22,6 +24,9 @@ window.metapi = {};
     console.error("Failed to fetch API field mappings from MS API:", jqXhr);
   });
 
+  $.getScript("https://raw.githubusercontent.com/joewalnes/reconnecting-websocket/f8055b77ba75e5d564ffb50d20a483bdd7edccdf/reconnecting-websocket.min.js",
```

Comment:
This probably won’t work, because GitHub serves it with a `Content-Type` of `text/plain`. [`https://cdn.rawgit.com/joewalnes/reconnecting-websocket/fd7c819b/reconnecting-websocket.js`](https://cdn.rawgit.com/joewalnes/reconnecting-websocket/fd7c819b/reconnecting-websocket.js) might be better. I also used the non-minified script because it’s newer than the minified one.